recipes/easytest/meta.yaml

```diff
@@ -0,0 +1,71 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "easytest" %}
+{% set version = "0.1.3" %}
+{% set sha256 = "bfecf6663b14bd5418e2d36b615ccdf7" %}
```

Comment:
This is actually the md5 key taken from pipit like described in the documentation.